### PR TITLE
fix(templates): Update `Microsoft.WindowsAppSDK` in templates

### PR DESCRIPTION
**Pull request by Youssef1313**
This avoids a NuGet error "detected package downgrade" when a reference to Uno.Material.WinUI is added.

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj [hunk 1]
```diff
@@ -20,32 +20,27 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221109.1" />
```
Comment (jeromelaban) [inline src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj:23]: Why did you remove this one? It comes transitively, but it's generally best to have the explicit dependency for users to upgrade.

File: src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj [hunk 2]
```diff
@@ -20,32 +20,27 @@
...
 		<PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
```
Comment (jeromelaban) [inline src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj:26]: We have logging initialization using this.

File: src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj [hunk 3]
```diff
@@ -20,32 +20,27 @@
...
-		<PackageReference Include="Uno.WinUI" Version="4.7.0-dev.417" />
```
Comment (jeromelaban) [inline src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj:27]: We need to keep this one, as we still have users referencing `VisibleBoundsPadding` and other controls provided by the `Uno.UI.Toolkit` assembly.

Comment (Youssef1313) [reply src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj:27]: I removed all those that come transitively to avoid maintaining versions in two projects (both for us and for template consumers), but I can add back.

Comment (jeromelaban) [reply src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj:27]: Right. That makes sense, at least it's not coming from the uno package itself, that should do.

**Review by jeromelaban — APPROVED** [review]
